dietpi/dietpi-drive_manager

```diff
@@ -1143,26 +1143,91 @@
 
 				if (( ${aDRIVE_ISMOUNTED[$INDEX_DRIVE_BEING_EDITED]} )); then
 
-					G_WHIP_MSG 'Partition must be unmounted, before check and repair can be run.\n\nPlease unmount the partition, then try again.'
+					G_WHIP_YESNO 'For safe check and repair, the drive needs to be unmounted.\n\nDo you want to continue with the drive being unmounted automatically?'
+					(( $? )) && TARGETMENUID=1 && return
+
+					# - disable swap
+					G_RUN_CMD swapoff -a
+
+					# - Unmount drive
+					G_RUN_CMD umount ${aDRIVE_MOUNT_SOURCE[$INDEX_DRIVE_BEING_EDITED]}
```

Comment:
Need to also remount the drive after checking?:
```
G_RUN_CMD mount -a
```

Reply:
I'll merge and take a look.

Reply:
Nope, it gets remounted automatically.